manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml

```diff
@@ -242,6 +242,8 @@ spec:
     image: "${OSSMCONSOLE_2_4}"
   - name: ossmconsole_v1_73
     image: "${OSSMCONSOLE_1_73}"
+  - name: operator
+    image: "${KIALI_OPERATOR}"
```

Comment:
see my comment in the other PR about this section - make this the very first image in the related Images section.